staging/src/k8s.io/apimachinery/pkg/api/validate/doc.go

```diff
@@ -32,6 +32,17 @@ limitations under the License.
 // a value in a list-map.  Validators which care about UPDATE operations should
 // look at the opCtx argument to know which operation is being executed.
 //
+// Tightened validation (also known as ratcheting validation) is supported by
```

Comment:
I think we will need to be really careful about using ratcheting, especially for things like pods, which can come from deployments, where the deployment might pass ratcheting (allowing the wider value) but the pod appears as a CREATE and thus fails.